reboot/reboot-gen

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
```

Comment:
Late to the game but it seems a bit excessive to have a generator script when all you need to do is:

```
DEVICE_TYPE="raspberrypi4"
mender-artifact write module-image \
    -T reboot \
    -t ${DEVICE_TYPE} \
    -o reboot.mender \
    -n reboot
```

Reply:
Thank you Mr. Krak. I thought it was a standard to provide a script. I can remove it if you want me to. Actually you will have to make me removeit.

Reply:
Yeah, somewhat standard for the more "complex" modules that require input files etc. In this case `mender-artifact` alone should work just fine as there is no additional error checking beside what is already provided by the `mender-artifact` tool.

Less code is always better. This is is not a priority, but if you wish to clean it up at some point I would be happy :)

Reply:
My vote would be to keep it, actually. I know it's kind of redundant, but I like that we have one, and only one, way to generate Update Module Artifacts. Easier to wrap your head around if you're new to the game.

Reply:
I would love to keep you both happy, but if I have to vote I would keep it as well. makes sense to me what you say.

Reply:
Ok, lets keep it then :+1: